drivers/rackspace/rackspace.go

```diff
@@ -62,6 +63,16 @@ func GetCreateFlags() []cli.Flag {
 			EnvVar: "OS_FLAVOR_ID",
 		},
 		cli.StringFlag{
+			Name:  "rackspace-net-id",
+			Usage: "Rackspace comma separated network ids the machine will be connected on",
```

Comment:
Hrm... A `StringFlag` isn't really meant to be used for specifying multiple options - would be better to use a `StringSliceFlag`, then the flag can be repeated a number of times (it's more in line with how other options work in docker-machine):

```go
...
    cli.StringSliceFlag{
        Name:  "rackspace-net-id",
        Usage: "Rackspace network id the machine will be connected on. Can be specified multiple times.",
        Value: &cli.StringSlice{},
    },
...
```